test/server_integration/http/platform/status.ts

```diff
@@ -23,6 +23,9 @@ export default function ({ getService }: FtrProviderContext) {
     return resp.body.status.plugins[pluginName];
   };
 
+  // max debounce of the status observable + 1
+  const statusPropagation = () => new Promise((resolve) => setTimeout(resolve, 501));
```

Comment:
I had a few CI failures while working on the PR, because `statusPluginB`'s status was not yet updated when we were accessing it. Adding a delay equivalent to the `debounce` interval we're using internally in the status observables did resolve the flakyness, but I just don't understand why this PR caused this, as it's not supposed to change anything to the status computation internals, or the associated delays...